easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CmdCp'
+
+name = 'FastTree'
+version = '2.1.11'
+
+homepage = 'http://www.microbesonline.org/fasttree/'
+description = """FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
+ or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
+ time and memory. """
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'openmp': True}
+
+# HTTPS cert error:
+# hostname 'www.microbesonline.org' doesn't match either of 'genomics.lbl.gov', 'mojave.qb3.berkeley.edu', ...
+source_urls = ['http://www.microbesonline.org/fasttree/']
+sources = ['%(name)s-%(version)s.c']
+checksums = ['9026ae550307374be92913d3098f8d44187d30bea07902b9dcbfb123eaa2050f']
+
+builddependencies = [('binutils', '2.35')]
+
+skipsteps = ['source']
```

Comment:
Do this https://github.com/easybuilders/easybuild-easyconfigs/pull/12794#issuecomment-830578384 here as well?

Reply:
done in b9ab9ccb35